Source/WebKit/WebProcess/WebPage/FindController.cpp

```diff
@@ -233,16 +233,12 @@ void FindController::updateFindUIAfterPageScroll(bool found, const String& strin
         hideFindIndicator();
 }
 
-void FindController::findString(const String& string, OptionSet<FindOptions> options, unsigned maxMatchCount, CompletionHandler<void(bool)>&& completionHandler)
+void FindController::findString(const String& string, OptionSet<FindOptions> options, unsigned maxMatchCount, TriggerImageAnalysis shouldTriggerImageAnalysis, CompletionHandler<void(bool)>&& completionHandler)
```

Comment:
Nit - I would maybe call this enum `CanTriggerImageAnalysis` instead, since whether we end up doing so depends on whether the feature flag is set.